File: py/mixpaneldatapipelines_sdk/utility/make_context.py
# MixpanelDataPipelines SDK utility: make_context

from mixpaneldatapipelines_sdk.core.context import MixpanelDataPipelinesContext


def make_context_util(ctxmap, basectx):
    return MixpanelDataPipelinesContext(ctxmap, basectx)

File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// MixpanelDataPipelines SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class MixpanelDataPipelinesMakeContext
{
    public static function call(array $ctxmap, ?MixpanelDataPipelinesContext $basectx): MixpanelDataPipelinesContext
    {
        return new MixpanelDataPipelinesContext($ctxmap, $basectx);
    }
}
